chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CHPLFileController.java

```diff
@@ -65,7 +65,7 @@ public CHPLFileController(final CHPLFileManager chplFileManager) {
     }
 
     @ApiOperation(value = "Retrieve an API Documentation file",
-            notes = "Retrieves the current API Documentation file.")
+            notes = "Retrieves the current API Documentation file.  Security Restrictions: ROLE_ADMIN or ROLE_ONC")
```

Comment:
This can't be right. The API Information Collections page calls this endpoint, and it's one of the public pages

Reply:
Was this supposed to be on the POST endpoint? The GET should be open to all

Reply:
That's probably why it was removed.  I did not have a recollection of why the security was removed.

Reply:
Addressed